components/formats-bsd/src/loci/formats/in/FakeReader.java

```diff
@@ -150,6 +152,10 @@ public class FakeReader extends FormatReader {
   /* physical sizes */
   private Length physicalSizeX, physicalSizeY, physicalSizeZ;
 
+  /* channel wavelengths */
+  ArrayList<Length> excitationWavelengths = new ArrayList<Length>();
+  ArrayList<Length> emissionWavelengths = new ArrayList<Length>();
```

Comment:
Contradicting myself a bit, just realized these two lines will invalidate memo files, which isn't suitable for a patch release. I'd imagine both could be made `transient` as they aren't needed in `openBytes`; otherwise we'd need to defer this until 8.2.0.